packages/react-native/lib/index.js

```diff
@@ -15,13 +15,63 @@ import {Platform, View} from 'react-native';
 import WebView from 'react-native-webview';
 import {WebviewEventHandler} from './message-handler';
 import AsyncStorage from '@react-native-async-storage/async-storage';
+import {AccountDetails} from '@docknetwork/wallet-sdk-core/lib/modules/account';
+import {DocumentType} from '@docknetwork/wallet-sdk-core/lib/types';
+
+export type WalletSDKContextProps = {
+  wallet: Wallet,
+  sdkStatus: string,
+};
 
 export const WalletSDKContext = React.createContext({
   wallet: null,
 });
 
 setStorage(AsyncStorage);
 
+export function getStorage() {
+  return AsyncStorage;
+}
+
+export const filterDocsByType = (type: DocumentType) => doc =>
+  doc.type === type;
+export const filterByIds = idList => doc => idList.find(v => v === doc.id);
+export const findDocument = (address, documents) =>
+  documents.filter(doc => doc.id === address)[0];
+export const findRelatedDocs = (document, documentList) =>
+  document && document.correlation
+    ? documentList.filter(doc => document.correlation.find(id => id === doc.id))
+    : [];
+
+function getAccount(address, documents): AccountDetails {
+  const addressDoc = findDocument(address, documents);
+
+  if (!addressDoc) {
+    return null;
+  }
+
+  const correlation = findRelatedDocs(addressDoc, documents);
+  const currencyDoc = correlation.find(filterDocsByType('Currency'));
```

Comment:
are we guaranteed that there will only ever be one `Currency` or `Mnemonic` doc? i.e. could `find` return an array?

Reply:
Yes, these document are generated in the account creation. Also, the `find` method will return only one item and should not be an array